runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -2695,12 +2695,20 @@ J9::CodeGenerator::processRelocations()
          }
 
       TR::SymbolValidationManager::SymbolValidationRecordList &validationRecords = self()->comp()->getSymbolValidationManager()->getValidationRecordList();
```

Comment:
I would like @mpirvu to review this specific commit to make sure the checking to enable/disable is done at the right place and in the right way.